functions/Errors/ErrorFmp.fmfn

```diff
@@ -25,4 +27,4 @@
  * =====================================
  */
 
-Error ( "fmp" ; Get ( LastError ) ; "" ; errorDetails )
+Error ( "fmp" ; Get ( LastError ) ; "" ; details )
```

Comment:
Had a discussion with @dansmith65 that raised up concerns about this function call clearing the internal JSON cache even when `Get ( LastError )` = 0.

I saw value in having a $error json object with an error code of 0 along with other details for some theoretical logging situation, but I'm starting to lean the other way because of the possible performance price.

The function would then do:
```
If ( Get ( LastError ) ;
     Error ( "fmp" ; Get ( LastError ) ; "" ; details ) ;

    false)
```

The false scenario could also involve a `let(  )` statement if we want to manually build a json payload into $error.

Looking for feedback about this concept.

Reply:
That pattern would be similar to the `ErrorIf (  )` pattern.

Reply:
FileMaker's handling of JSON is pretty bad and it's always nice to support a developer's desire to use an alternative.